cli/cmd/tapRunner.go

```diff
@@ -257,19 +253,15 @@ func watchPodsForTapping(ctx context.Context, kubernetesProvider *kubernetes.Pro
 		}
 	}
 	restartTappersDebouncer := debounce.NewDebouncer(updateTappersDelay, restartTappers)
-	timer := time.AfterFunc(time.Second*10, func() {
-		mizu.Log.Debugf("Waiting for URL...")
+
+	go func() {
```

Comment:
this one should be moved after the first call to "updateCurrentlyTappedPods"

Reply:
fixed